### ISSUE TITLE
Translation

### ISSUE THREAD

**Issue by eson57** [skvark/Textractor#10]
Hi,

I really like the potential of this app, and would like it even more if I could have it in my native language. I took a look at the .ts file and found that it is far from complete. Are you planning to update translation file and implement localisation? If so, I would be the first to translate.
Great job anyway. Thanks!

**Comment by skvark** [comment]
Yes, it's intentionally left incomplete because of limited dev time. I'll update it hopefully soon and provide a transifex project to ease the translation process.

**Comment by eson57** [comment]
Ok, thanks. I'll watch this project so please make a note somewhere, when transifex project is ready.
